back/engines/commercial/report_builder/app/services/report_builder/queries/survey_results.rb

```diff
@@ -1,5 +1,5 @@
 module ReportBuilder
-  class Queries::SurveyResults
+  class Queries::SurveyResults < ReportBuilder::Queries::Base
```

Comment:
Because we're going to use the single survey question instead of this survey results endpoint, we might not need this query. We might need to remove the current survey results widget altogether since it doesn't properly allow hiding questions.

Reply:
Or do you think there's still a good reason to keep that widget?

Reply:
Indeed, if we introduce this new single survey question, we can definitely remove the old one. Until then, maybe we could keep it as it doesn't hinder.